nistats/contrasts.py

```diff
@@ -276,3 +276,71 @@ def __rmul__(self, scalar):
 
     def __div__(self, scalar):
         return self.__rmul__(1 / float(scalar))
+
+
+def fixed_effects_img(contrast_imgs, variance_imgs, mask,
+                         precision_weighted=False):
+    """Compute the fixed effets given images of effects and variance
+
+    Parameters
+    ----------
+    contrast_imgs: list of Nifti1Images or strings
+              the input contrast images
+    variance_imgs: list of Nifti1Images or strings
+              the input variance images
+    mask: Nifti1Image or NiftiMasker instance,
```

Comment:
It would be nice if this were an optional argument, and if a mask is not provided, that the mask can be learned like in `SecondLevelModel`. 

Otherwise, if the function that does the mask learning in `SecondLevelModel` can be abstracted to a more general function, that would make it easier to use that function prior to this one.